### ISSUE TITLE
[FR] BABYSTEPPING for Core Kinematics

### ISSUE THREAD

**Issue by ruggb** [MarlinFirmware/Marlin#10064]
I loaded 12b697c and everything was "perfect" until I started printing and realized babystepping Z is no longer working......9939 fixed the Y endstop annihilator but.........
Nothing was changed in the config.
Menu is on LCD; dbl click to BS works; turning the knob increases/decreases the numbers but nothing happens.

**Comment by thinkyhead** [comment]
Actually, `BABYSTEPPING` has never been fully compatible with CORE kinematics. But for CoreXY baby-stepping for Z should work, since it happens to be the same as regular Cartesian Z axis.

So try setting `MINIMUM_STEPPER_PULSE` to 2 and see if it starts to work.

**Comment by thinkyhead** [comment]
I've now added an implementation for Core-compatible `BABYSTEPPING` in [this branch](https://github.com/thinkyhead/Marlin/tree/bf1_core_babystepping) so that you can enable `BABYSTEP_XY` if you want. Give it a try and see if it does the correct movement in those axes.

**Comment by ruggb** [comment]
Does this commit have the Y fix??? I'd rather not move off the BF branch I am on - I confuse easily.........and I don't need BS XY. I never use it. It never worked for Core but I don't have a need for it either. It might be useful on another machine, but it probably worked for those.
Prior to the Y fix babystepping Z worked fine. I didn't change any BS settings. BS Z just stopped working with the Y fix.

**Comment by xelopics** [comment]
babysteping works by me not too i have mks sbase on coreXY Custom Build with Marlin 2.0

**Comment by thinkyhead** [comment]
> babysteping works by me not too

"Babystepping doesn't work for me either"?

**Comment by thinkyhead** [comment]
> Does this commit have the Y fix???

Do you mean "Z fix"? There is no Z fix needed that I can see. Babystepping Z on a CoreXY uses the same technique as a regular Cartesian.

Try increasing `MINIMUM_STEPPER_PULSE` as suggested and see if it helps.

**Comment by thinkyhead** [comment]
>  I don't need BS XY. I never use it.

No problem, we can find another Core owner to test this for us.

**Comment by ruggb** [comment]
When are you going to get BS Z working again???????? 4dad862 doesn't work either.
Everything looks like it is working except there is no movement of the Z axis when I turn the display knob.
When that happens I'll try out XY when you put it into the bugfix branch I am using.
But I still see no need for it on a core machine.

**Comment by thinkyhead** [comment]
> When are you going to get BS Z working again????????

That's really obnoxious and childish. If you want help with this you need to chill out.

**Comment by thinkyhead** [comment]
Did you try increasing `MINIMUM_STEPPER_PULSE` as I've repeatedly suggested?
You haven't said one way or the other. You simply keep repeating yourself.

I'm starting to think your whole aim is to just troll us.

**Comment by ruggb** [comment]
@thinkyhead I went back and reloaded each of the loads. BS Z started working again - I have no idea why. Nothing changed. The MINIMUM_STEPPER_PULSE was set to 2 by default but that is specified for 8825 drivers which I have. I didn't change it (I don't think I did anyway). But it definitely wasn't functioning previously. I can't imagine a H/W issue as it printed fine. If it quits again, I'll dig into it.
I also loaded ed18791 and the BS X&Y appear to work though they are not available until the print starts. You might want to change that but don't complicate BS Z double click. I can't imagine why one would change X&Y on a Core machine during a print. If used, it is probably only for testing.

Troll indeed! I have nothing better to do.
